tests/end_to_end/test_helper.cpp

```diff
@@ -455,9 +455,15 @@ Glue_close(int file_descriptor)
         return -1;
     } else {
         bool found = _duplicate_handles.dereference_if_found(it->second);
-        if (!found)
+        if (!found) {
             // No duplicates. Close the handle.
-            REQUIRE((ebpf_fuzzing_enabled || ebpf_api_close_handle(it->second) == EBPF_SUCCESS));
+            ebpf_result_t result = ebpf_api_close_handle(it->second);
+            if (ebpf_fuzzing_enabled) {
+                (void)result;
+            } else {
+                REQUIRE(result == EBPF_SUCCESS);
+            }
+        }
```

Comment:
nit: can we simply swap the order?

`REQUIRE((ebpf_api_close_handle(it->second) == EBPF_SUCCESS || ebpf_fuzzing_enabled))`